sound/hybrid_font.h

```diff
@@ -234,11 +234,38 @@ class HybridFont : public SaberBase, public Looper {
     EnableAmplifier();
     if (!f->files_found()) return RefPtr<BufferedWavPlayer>(nullptr);
     RefPtr<BufferedWavPlayer> player = GetFreeWavPlayer();
-    if (player) {
-      player->set_volume_now(font_config.volEff / 16.0f);
-      player->PlayOnce(f);
-      current_effect_length_ = player->length();
+    if (!player) {
+      STDOUT.println("Out of WAV players! Getting more...");
+      float oldest_ = 0;
```

Comment:
kill_ and oldest_ are local variables, they should not end with _.
They should also have better names, like maybe "highest_progress" and "player_to_restart".

Reply:
sure.